### PR TITLE
Leaflet 1.0 announcement blog post

### PR DESCRIPTION
**Pull request by mourner**
A draft of 1.0 announcement, any edits / additions?
@IvanSanchez @yohanboniface @perliedman @danzel

### CODE REVIEW

**Comment by mourner** [comment]
I feel like this needs a gif.

File: docs/_posts/2016-09-27-leaflet-1.0-final.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: The wait is over. Meet Leaflet 1.0.
```
Comment (IvanSanchez) [inline docs/_posts/2016-09-27-leaflet-1.0-final.md]: This sounds so much like an apple commercial, I'll ask our in-house designer to throw some meme ideas.

File: docs/_posts/2016-09-27-leaflet-1.0-final.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+---
+
+Today marks the final release of Leaflet 1.0. It's a culmination of several years of work by dozens of contributors from all over the world. This is the fastest, most stable and polished Leaflet release ever.
+
+[The changelog](https://github.com/Leaflet/Leaflet/blob/master/CHANGELOG.md) contains _over 400 changes_ compared to the previous stable, v0.7.7. The highlights include:
+
+- Huge performance improvements in all aspects of the library and vector layers in particular.
+- New `L.Tooltip` class for all your dynamic labeling needs.
+- Flyover animations (zooming and panning in a curve with `map.flyTo`)
+- Fractional zoom level support (`map.setZoom(12.34`).
```
Comment (IvanSanchez) [inline docs/_posts/2016-09-27-leaflet-1.0-final.md]: Missing parenthesis here

File: docs/_posts/2016-09-27-leaflet-1.0-final.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+- Much better tile loading algorithm with less flickering.
+- Custom pane management (including multiple vector layer panes and interleaving vectors and tile layers).
+- Better support for non-standard projections.
+- More accessibility features.
+- Vastly improved documentation.
+- _Tons_ of bugfixes and stability improvements.
+
+This release wouldn't be possible without such an amazing, diverse, loving community. We're incredibly proud to be a part of it. Thank you very much for your kind support!
+
+![](http://www.reactiongifs.com/r/msy.gif)
```
Comment (IvanSanchez) [inline docs/_posts/2016-09-27-leaflet-1.0-final.md]: We used this GIF already in a previous beta release blog post. Should be changed.

File: docs/_posts/2016-09-27-leaflet-1.0-final.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Starting with 1.0, we're switching to a fast release cycle, where a new version will come out every 2–4 weeks. We'll also prefer many small, incremental changes (with releases in between) over ambitious rewriting quests.
+
+### What's next?
+
+In addition to supporting the new release with fixes and improvements, the following areas will now get our close attention:
+
+- A cleaner, more stable plugin ecosystem. We want to make sure that all popular plugins work well with the latest release and are maintained. We'll revise our plugin list page, tighten our review process, take more key plugins under the wing of the Leaflet organization on GitHub and spend more time working on them.
+- Tutorials and examples to cover more advanced features of the library, its popular plugins, and integrations with libraries like D3 and React/Angular.
+- Explore switching to a more modern JavaScript flavor and module system (ES6, Babel, Rollup).
```
Comment (IvanSanchez) [inline docs/_posts/2016-09-27-leaflet-1.0-final.md]: I'd replace `Babel` with `~~Babel~~ Bublé`, just for added fun :-D

File: docs/_posts/2016-09-27-leaflet-1.0-final.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+- Better support for non-standard projections.
+- More accessibility features.
+- Vastly improved documentation.
+- _Tons_ of bugfixes and stability improvements.
+
+This release wouldn't be possible without such an amazing, diverse, loving community. We're incredibly proud to be a part of it. Thank you very much for your kind support!
+
+### New release cycle
+
+This release is a way overdue and has taken too long. In part this can be attributed to our initial ambition for 1.0 that required rewriting many parts of the library almost from scratch, in part due to perfectionism (some of the beta/rc releases were already more stable than our last stable release), and in part because we undersestimated the cost of introducing big breaking changes in a library used by thousands of developers.
```
Comment (theashyster) [inline docs/_posts/2016-09-27-leaflet-1.0-final.md]: undersestimated -> underestimated

File: docs/_posts/2016-09-27-leaflet-1.0-final.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+authorsite: http://agafonkin.com/en
+---
+
+Today marks the final release of Leaflet 1.0. It's a culmination of several years of work by dozens of contributors from all over the world. This is the fastest, most stable and polished Leaflet release ever.
+
+[The changelog](https://github.com/Leaflet/Leaflet/blob/master/CHANGELOG.md) contains _over 400 changes_ compared to the previous stable, v0.7.7. The highlights include:
+
+- Huge performance improvements in all aspects of the library and vector layers in particular.
+- New `L.Tooltip` class for all your dynamic labeling needs.
+- Flyover animations (zooming and panning in a curve with `map.flyTo`)
```
Comment (theashyster) [inline docs/_posts/2016-09-27-leaflet-1.0-final.md]: This is the only bullet point missing a `.` at the end of it

**Comment by danzel** [comment]
👍

**Pull request merged**